python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -702,10 +724,13 @@ def _runStreakMasking(self, difference):
                                              self.config.streakBinFactor)
         binnedExposure = afwImage.ExposureF(binnedMaskedImage.getBBox())
         binnedExposure.setMaskedImage(binnedMaskedImage)
-        binnedExposure.setPsf(difference.psf)  # exposure must have a PSF
+        # Clear the DETECTED mask plane before streak detection
+        binnedExposure.mask &= ~binnedExposure.mask.getPlaneBitMask('DETECTED')
         # Rerun detection to set the DETECTED mask plane on binnedExposure
-        _table = afwTable.SourceTable.make(self.schema)
-        self.detection.run(table=_table, exposure=binnedExposure, doSmooth=True)
+        sigma = difference.psf.computeShape(difference.psf.getAveragePosition()).getDeterminantRadius()
+        _table = afwTable.SourceTable.make(afwTable.SourceTable.makeMinimalSchema())
```

Comment:
Do you specifically not want the `self.schema` we used previously that had some extra columns floating around? I assumed it was more efficient to pass along an existing schema than making a new one from scratch, but it probably doesn't matter much either way.

Reply:
The minimal schema is trivial to create, and makes detection slightly faster since the resulting table has far fewer columns.